### ISSUE TITLE
Not possible to convert Vk* to vk::* in 32-bit

### ISSUE THREAD

**Issue by spfuetzner** [KhronosGroup/Vulkan-Hpp#70]
I know that VULKAN_HPP_TYPESAFE_CONVERSION should enable this but I think there is a more elegant solution. Instead of disabling the conversion constructor, just make it explicit if VULKAN_HPP_TYPESAFE_CONVERSION is not defined. This prevents unwanted conversions but allows explicit construction. Otherwise I always have to define the macro to make the following example possible:
```
vk::SurfaceKHR f()
{
	VkSurfaceKHR surf_ {};
	auto const result {glfwCreateWindowSurface(inst, wnd, nullptr, &surf_)};
	return vk::SurfaceKHR {surf_};
}
```
It should always be possible to construct the nice vk::* objects from raw Vk* handles. This could stay implicit for 64-bit because it is type safe. It should not be mandatory to give up type safety in 32-bit by always defining the macro.

**Comment by mtavenrath** [comment]
@spfuetzner I fully agree. We're missing explicit constructors for Vk* handles and will add them.

**Comment by asuessenbach** [comment]
Resolved with pull-request #82.
